### PR TITLE
MAHOUT-2155 Add Javadocs for RandomWrapper

### PR DESCRIPTION
**Pull request by rawkintrevo**
### Purpose of PR:

Add javadocs for RandomWrapper

### Important ToDos
Please mark each with an "x"
- [x] A JIRA ticket exists (if not, please create this first)[https://issues.apache.org/jira/browse/mahout/]
- [x] Title of PR is "MAHOUT-XXXX Brief Description of Changes" where XXXX is the JIRA number.
- [x] Assigned JIRA to self
- [x] Added documentation in scala docs/java docs, and to website

Does this change break earlier versions? No

Is this the beginning of a larger project for which a feature branch should be made? No

### CODE REVIEW

File: core/src/main/java/org/apache/mahout/common/RandomWrapper.java [hunk 1]
```diff
@@ -47,14 +65,21 @@ public void setSeed(long seed) {
...
   }
 
+    /**
+     * @return the delegate random number generator.
+     */
   public RandomGenerator getRandomGenerator() {
     return random;
   }
 
+
```
Comment (andrewmusselman) [inline core/src/main/java/org/apache/mahout/common/RandomWrapper.java:82]: Do we need this extra newline?

File: core/src/main/java/org/apache/mahout/common/RandomWrapper.java [hunk 2]
```diff
@@ -47,14 +65,21 @@ public void setSeed(long seed) {
...
   @Override
   protected int next(int bits) {
```
Comment (andrewmusselman) [inline core/src/main/java/org/apache/mahout/common/RandomWrapper.java:84]: While we're at it can we add a comment block to this method?

**Review by andrewmusselman — APPROVED** [review]
Looks good with changes suggested

File: core/src/main/java/org/apache/mahout/common/RandomWrapper.java (hunk 2 above)
Comment (rawkintrevo) [reply core/src/main/java/org/apache/mahout/common/RandomWrapper.java:92]: Addressed.

**Pull request merged**